commons/commons-serialization/src/main/kotlin/ac/uk/ebi/biostd/tsv/serialization/TsvBuilder.kt

```diff
@@ -1,18 +1,26 @@
 package ac.uk.ebi.biostd.tsv.serialization
 
+import ebi.ac.uk.base.EMPTY
+
 internal const val TITLE_KEY = "Title"
 internal const val ACC_NO_KEY = "Submission"
 internal const val ROOT_PATH_KEY = "RootPath"
+internal val specialCharRegex = "[\n\t\"]".toRegex()
```

Comment:
I think we could rename this to MULTILNE_BLOCK or something clearer because the current name is very generic

Reply:
the issue here is that not only multiline block but also a block for example with a inner tab